public/js/showoff.js

```diff
@@ -1274,16 +1268,5 @@ function setupStats()
 
 /* Is this a mobile device? */
 function mobile() {
-/*
-  return ( navigator.userAgent.match(/Android/i)
-            || navigator.userAgent.match(/webOS/i)
-            || navigator.userAgent.match(/iPhone/i)
-            || navigator.userAgent.match(/iPad/i)
-            || navigator.userAgent.match(/iPod/i)
-            || navigator.userAgent.match(/BlackBerry/i)
-            || navigator.userAgent.match(/Windows Phone/i)
-  );
-*/
-
   return ( $(window).width() <= 480 )
```

Comment:
It's only used in one spot, to determine if scrollIntoView should be used on the presenter's table of contents.  I'm wondering if there is a way to do this in CSS or if it is even necessary.